### ISSUE TITLE
Option to export numeric arrays all-on-one-line

### ISSUE THREAD

**Issue by francislavoie** [brick/varexporter#3]
I'm really liking this lib!

For my use-case, I have large numeric (non-assoc) arrays of numbers that I'm exporting. I'd like an option that would avoid new-lines + indentation for non-numeric arrays to save a ton of lines in the output.

An option to make numeric arrays stay on one line and keep associative arrays multi-line would be the ideal output for me.

Thanks for your work on this!

**Comment by francislavoie** [comment]
Just as a proof-of-concept, I hacked it together myself in `GenericExporter.php`

https://gist.github.com/francislavoie/023f7abe091e21d27eb51d70c1ebca1a

I'm not sure if this is viable or not, I just wanted to see how it would look.

**Comment by BenMorel** [comment]
Hi, this is a legitimate use case, I actually had such a requirement myself to export data in another lib and keep it readable. So if I understand correctly, the idea would be to export inline if the array:

- is numeric
- contains only scalar values

This could work. I'm not sure if this should be an option or the default, though.

**Comment by francislavoie** [comment]
IMO up to you if you want to make it a default, but I think it should at least still be a feature flag. The current behavior closely matches var_export and might be preferred that way for some.

**Comment by BenMorel** [comment]
Implemented in 066bc3e7f1c47a98004ec6f0b1de131659699576 and temporarily available here:
https://github.com/brick/varexporter/tree/inline-numeric-scalar-array

```php
VarExporter::export([
    'one' => ['hello', 'world', 123, true, false, null, 7.5],
    'two' => ['hello', 'world', ['one', 'two', 'three']]
], VarExporter::INLINE_NUMERIC_SCALAR_ARRAY);
```

```php
[
    'one' => ['hello', 'world', 123, true, false, null, 7.5],
    'two' => [
        'hello',
        'world',
        ['one', 'two', 'three']
    ]
]
```

Can you please check if this is what you expect, before I tag a release?

**Comment by francislavoie** [comment]
Yup, LGTM! Thank you!!

**Comment by BenMorel** [comment]
Released as [0.2.1](https://github.com/brick/varexporter/releases/tag/0.2.1).

Thanks for the suggestion! 👍
